tests/smoke.bats

```diff
@@ -0,0 +1,38 @@
+@test "Test home page is served" {
+    run curl -o /dev/null -L -s -w "%{http_code}\n" $TARGET_URL
```

Comment:
Do you mind using the long-format for these options? Easier to read and we're not trying to save space here :smile: